<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_perso</class>
 <widget class="QDialog" name="Dialog_perso">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane { /* The tab widget frame */
border-top: 2px solid #2a2a2a; 
border-left: 2px solid #2a2a2a;
border-right: 2px solid #2a2a2a ; 
border-bottom: 2px solid #2a2a2a; 
background: #dcdcdc;
	
	
	/*border-top-color: rgb(0, 0, 0);*/

}
QTabWidget::tab-bar {
left: 0px; /* move to the right by 5px */
}
QTabBar::tab {
background: #dcdcdc;
border: 2px solid #535353;
border-bottom-color: #535353; 
border-top-left-radius: 7px;
border-top-right-radius: 7px;
min-width: 80px;
padding: 8px;
color: rgb(50, 50, 162);
	color: rgb(0, 0, 0);
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:selected,QTabBar::tab:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
}
QTabBar::tab:selected {
border-color: #9B9B9B;
border-bottom-color: #535353; /* same as pane color */
color: #dcdcdc;
font-size:13px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:!selected {
margin-top: 2px; /* make non-selected tabs look smaller */
}</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>acceuil</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>561</width>
       <height>371</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTabWidget::pane { /* The tab widget frame */
border-top: 2px solid #2a2a2a; 
border-left: 2px solid #2a2a2a;
border-right: 2px solid #2a2a2a ; 
border-bottom: 2px solid #2a2a2a; 
background: #dcdcdc;
	
	
	/*border-top-color: rgb(0, 0, 0);*/

}
QTabWidget::tab-bar {
left: 0px; /* move to the right by 5px */
}
QTabBar::tab {
background: #dcdcdc;
border: 2px solid #535353;
border-bottom-color: #535353; 
border-top-left-radius: 7px;
border-top-right-radius: 7px;
min-width: 80px;
padding: 8px;
color: rgb(50, 50, 162);
	color: rgb(0, 0, 0);
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:selected,QTabBar::tab:hover {
	
	
	color: rgb(255, 255, 255);
	background-color: rgb(0, 0, 127);
}
QTabBar::tab:selected {
border-color: #9B9B9B;
border-bottom-color: #535353; /* same as pane color */
color: #dcdcdc;
font-size:13px;
font-family:Arial, Helvetica, sans-serif;
}
QTabBar::tab:!selected {
margin-top: 2px; /* make non-selected tabs look smaller */
}</string>
     </property>
     <property name="title">
      <string>Personnel</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}
</string>
      </property>
      <property name="text">
       <string>nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="aaaaa">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}
</string>
      </property>
      <property name="text">
       <string>prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}
</string>
      </property>
      <property name="text">
       <string>cin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}
</string>
      </property>
      <property name="text">
       <string>age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>190</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}
</string>
      </property>
      <property name="text">
       <string>salaire</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_cin">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_age">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_salaire">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>190</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
background-color: rgb(0, 0, 127);
border: none;
}
</string>
      </property>
      <property name="text">
       <string>Numerro tel</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_numero">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>230</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="imagee">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>40</y>
        <width>141</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="insererphoto">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>200</y>
        <width>111</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
color: #dcdcdc;
background:#535353;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
      </property>
      <property name="text">
       <string>inserer photo</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1200</width>
       <height>660</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>PP.jpg</pixmap>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_ajouter">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_modifier">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_supprimer">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>420</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QComboBox" name="table_cin">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>420</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#FFC0CB;
background-color: rgb(0rgb(0, 255, 0), 0, 127);
border: none;
}
</string>
     </property>
    </widget>
    <zorder>label_6</zorder>
    <zorder>groupBox</zorder>
    <zorder>pb_ajouter</zorder>
    <zorder>pb_modifier</zorder>
    <zorder>pb_supprimer</zorder>
    <zorder>table_cin</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>afficher</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1200</width>
       <height>660</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>PP.jpg</pixmap>
     </property>
    </widget>
    <widget class="QTableView" name="tab_personnel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>771</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="trier_cin">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>410</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
     </property>
     <property name="text">
      <string>trier par cin</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trier_nom">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>410</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
     </property>
     <property name="text">
      <string>trier par nom</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trier_salaire">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>410</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
     </property>
     <property name="text">
      <string>trier par salaire</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Historique</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1200</width>
       <height>660</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>PP.jpg</pixmap>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>621</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="Suprimer_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>390</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
color: #000000;
background:#FFC0CB;
border: none;
font-size:12px;
font-family:Arial, Helvetica, sans-serif;
border: 2px solid ;
border-radius: 8px;
}
QPushButton:hover {
color:#fff;
	background-color: rgb(0, 0, 127);
border: none;
}</string>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="Quitter">
   <property name="geometry">
    <rect>
     <x>1108</x>
     <y>0</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
border-radius:20px;
background-color: rgb(56, 56, 56);
color: rgb(255, 255, 255);
}
QPushButton:hover
{
color: rgb(56, 56, 56);
background-color: rgb(255, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Smart Parc</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
